test/unit-tests/host-info.ts

```diff
@@ -0,0 +1,96 @@
+import { Yok } from "../../yok";
+import { HostInfo } from "../../host-info";
+import { ErrorsStub, CommonLoggerStub } from "./stubs";
+import { assert } from "chai";
+
+// Use custom class as isDarwin has only getter in HostInfo, while for current tests we need to set it to true or false.
+class DarwinHostInfo extends HostInfo {
```

Comment:
Maybe this should be `MockHostInfo` or something similar.
Currently it's a little bit strange. If the name is `DarwinHostInfo` I'm expecting
```
public get isDarwin(): boolean {
     return true;
}
```